docs/sdk/serverless-signing/conditional-signing.md

```diff
@@ -40,8 +40,11 @@ const go = async () => {
     return;
   }
 
+  const message = new Uint8Array(
```

Comment:
it looks like we're double hashing here, which is unnecessary.  we're SHA256-ing it, and then keccak256-ing it again.